features/works/work_create.feature

```diff
@@ -197,6 +197,45 @@ Feature: Create Works
       And I should see "Chapter"
       And I should see "1/?"
 
+  Scenario: Creating a new work in a new series with some invalid things should return to the new work page with an error message and series information still filled in
+    Given basic tags
+      And the following activated users exist
```

Comment:
You should be able to remove this line and the line after it now that coauthor and badcoauthor have been removed.